frontend/public/components/monitoring/receiver-forms/slack-receiver-form.tsx

```diff
@@ -229,6 +232,44 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
                 {t('slack-receiver-form~Find and link channel names and usernames.')}
               </div>
             </div>
+            <div className="form-group">
+              <label className="control-label" htmlFor="slack-title">
+                {t('slack-receiver-form~Title')}
+              </label>
+              <TextArea
+                id="slack-title"
+                aria-describedby="slack-title-help"
+                onChange={(value) =>
+                  dispatchFormChange({
+                    type: 'setFormValues',
+                    payload: { slack_title: value },
+                  })
+                }
+                value={formValues.slack_title}
+              />
+              <div className="help-block" id="slack-title-help">
+                {t('slack-receiver-form~The title of the slack message.')}
```

Comment:
We should probably capitalize Slack throughout this form if it's not already.

```suggestion
                {t('slack-receiver-form~The title of the Slack message.')}
```

Reply:
fixed